tests/ttnn/CMakeLists.txt

```diff
@@ -7,6 +7,9 @@ function(setup_ttnn_test_target target_name)
             ttnn
             Metalium::Metal
             GTest::gmock_main
+        PRIVATE
+            Boost::asio
+            Boost::lockfree
```

Comment:
I think this is the wrong place to add this. None of the other ttnn test targets require these libraries, so they should be added directly to `test_ccl_multi_cq_multi_device` like
```
target_link_libraries(test_ccl_multi_cq_multi_device PRIVATE Boost::asio Boost::lockfree)
```
rather than within `setup_ttnn_test_target`.